tests/featured-speaker.spec.ts

```diff
@@ -24,10 +24,19 @@ import { expect, test } from '@playwright/test';
 ].forEach(({ fullName, tagLine, profileUrl }) => {
     test(`Should provide link to page of featured speaker ${fullName} on home page`, async ({ page }) => {
         await page.goto('/');
+
         const featuredSpeakersSection = page.getByRole('region', { name: 'Featured Speakers' });
         const featuredSpeakersMenu = featuredSpeakersSection.getByRole('menu', { name: 'Featured Speakers' });
         const featuredSpeakersMenuItem = featuredSpeakersMenu.getByRole('menuitem', { name: fullName });
-        await expect(featuredSpeakersMenuItem.getByText(fullName)).toBeVisible();
+
+        // The Playwright's strict mode throws an error when multiple elements
```

Comment:
All this text seems a good reason to clean this up a little bit 😅. Not sure why I implemented it as `<span class="speaker-card__name">{{ $input.name }}</span>`, but from a user perspective, I would say it's just a header. `<h1 class="heading">{{ $input.name }}</h1>` results in the exact same layout. This expectation can easily be implemented as test (`featuredSpeakersMenu.getByRole('heading', { name: fullName })`, `featuredSpeakersMenu.getByRole('paragraph', { name: tagline })`, etc).

Reply:
done